### PR TITLE
Development

### PR DESCRIPTION
**Pull request by BenjaminGarza**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Great start to this project. But some few changes need to be made

File: main.css [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    
+    width:1440px;
+    height: 100%;
+    margin: auto;
+    
+}
+#main {
+  border: .01px transparent solid;
+  width: 1440px;
+  height:  805px;
```
Comment (Oghenebrume50) [inline main.css]: This line is affecting your code, the footer jumps up. You may want to look into heights and width, most times you don't need to explicitly set heights of a large container. and your width is off too, it leaves a big space that is scrollable, too big to be ignored.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <title>Document</title>
+</head>
+<body>
+   
+    <main id="main">
+       <article id="billboard">
```
Comment (Oghenebrume50) [inline index.html]: This article tag shouldn't be wrapping the nav tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+                        </div>
+                 </a>
+                <a  class="watch-film" href="">
+                       <div id="watch-film-container"></div>
+                </a>
+
+           </section>
+       </article>
+        <aside>
```
Comment (Oghenebrume50) [inline index.html]: The content in this aside tag should be in the footer, and the images should be passed as background images, just like you did for the ones above. use a div and pass as background image

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Fix those things and submit for re-review, nice attempt, you did really good.

**Comment by Oghenebrume50** [comment]
Hi, it is best to use positioning for inner elements and not outer container currently this is how your project displays

**Tips**
Create an outer container to hold nav, main and footer Set a display: flex on it
Flex: direction column
avoid using margins on the nav, main, footer.
style the inner elements

File: main.css [hunk 4]
```diff
@@ -32,9 +31,10 @@ width: 100%;
 
 
 nav{
-    
+    position: relative;
+    top: 230px;
     width: 62%;
-    height: 5%;
+    height: 30px;
     margin: 60px;
```
Comment (Oghenebrume50) [inline main.css]: If you remove this margin you would see how it affects the page display. You have to be careful with margins

File: main.css [hunk 5]
```diff
@@ -0,0 +1,266 @@
+body{
+    
+    width:1440px;
```
Comment (Oghenebrume50) [inline main.css]: You shouldn't set an explicit width like this on the body.

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]

**Comment by samgaco** [comment]
Hi, @BenjaminGarza I can't see any changes after the last review. Is everything ok?. I wait for your reply.

**Comment by BenjaminGarza** [comment]
> Hi, @BenjaminGarza I can't see any changes after the last review. Is everything ok?. I wait for your reply.

Just made more changes today and putting in for another code review, thanks!

**Comment by bolah2009** [comment]
Currently under review

**Review by bolah2009 — APPROVED** [review]
## Nice Job [Ben](../../)  :clap: 

The [clone](https://raw.githack.com/BenjaminGarza/Backgrounds-and-Gradients/Development/index.html) looks good to me. :+1:

#### Consider checking these [errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FBenjaminGarza%2FBackgrounds-and-Gradients%2FDevelopment%2Findex.html) before merging.

#### Friendly reminder: You should use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in this project. However, changing it now would take too much time. Please make sure you use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in the next project. You will find instructions in the Pathwright lesson description.

Good luck with your next project.

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**